### PR TITLE
cli: Improve usability and maintainability

### PR DESCRIPTION
**Pull request by netoax**
What does this implement/fix? Explain your changes.
---------------------------------------------------

This PR adds the following changes:

- Format code with prettier;
- Update options handling (`amqp`, `http`);
- Bump knot-cloud-sdk-js to v4.0.0;
- Add examples to the commands.

Any relevant logs, error output, etc?
-------------------------------------
(If it’s long, please paste to https://pastebin.com/ and insert the link here.)

Any other comments?
-------------------
...

Where has this been tested?
---------------------------
(Describe your environment setup here.)

**Operating System/Platform:** ubuntu 18.04

**NPM Version:** v6.14.15

**NodeJS Version:** v14

(Add more information here if needed.)

### CODE REVIEW

File: src/cmds/cloud/setData.js [hunk 1]
```diff
@@ -39,11 +41,13 @@ yargs
           describe: 'ID of the sensor to be updated',
         })
         .positional('value', {
-          describe: 'Value to set the sensor to. Supported types: boolean, number or Base64 strings.',
+          describe:
+            'Value to set the sensor to. Supported types: boolean, number or Base64 strings.',
           coerce: (value) => {
-            if (isNaN(value)) { // eslint-disable-line no-restricted-globals
+            if (isNaN(value)) {
+              // eslint-disable-line no-restricted-globals
```
Comment (netoax) [inline src/cmds/cloud/setData.js]: Remove this inline disable.

File: src/cmds/cloud/updateConfig.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const updateConfig = async (args) => {
 };
 
 yargs
-  .config('credentials-file', path => getFileCredentials(path))
+  .config('credentials-file', (path) => getFileCredentials(path))
   // .config('config-file', path => getFileConfig(path))
```
Comment (netoax) [inline src/cmds/cloud/updateConfig.js]: Why is this commented?

File: src/cmds/cloud/publishData.js [hunk 3]
```diff
@@ -35,9 +35,10 @@ yargs
         .positional('value', {
           describe: 'Value to be published. Supported types: boolean, number or Base64 strings.',
           coerce: (value) => {
-            if (isNaN(value)) { // eslint-disable-line no-restricted-globals
+            if (isNaN(value)) {
+              // eslint-disable-line no-restricted-globals
```
Comment (netoax) [inline src/cmds/cloud/publishData.js]: Same here.

**Review by esdrasjnr — APPROVED** [review]
Awesome patch 💯

**Review by lcbm — APPROVED** [review]
LGTM :smile:

**Pull request merged**